promptsource/templates/openai_humaneval/templates.yaml

```diff
@@ -0,0 +1,39 @@
+dataset: openai_humaneval
+templates:
+  4a108b1c-7514-488f-99ed-3ca5da70e103: !Template
+    answer_choices: null
+    id: 4a108b1c-7514-488f-99ed-3ca5da70e103
+    jinja: '{{ prompt }}
+
+      Given the following docstring, what is the function body?
```

Comment:
This should probably be "GIven the above"; Anyways this prompt is bad imo as there's a function name + a docstring above.

Would be better to have e.g. "Complete the following function\n{{ prompt }}" imo